libs/blocks/global-navigation/blocks/profile/profile.css

```diff
@@ -0,0 +1,133 @@
+:root {
+  --border-color: #e1e1e1;
+}
+
+.gnav-profile .gnav-profile-menu {
+  display: none;
+  position: fixed;
```

Comment:
Why `fixed` and not `absolute`? Say that the nav wrapper has a maximum width of 1200px, but the user has a wide monitor, say 3600px. Then the profile dropdown will be completely disconnected from its trigger/avatar.

Later edit: I saw that this changes to `absolute` on desktop devices, but I'm still not convinced this approach is the best. I know the styles are copied from the previous Milo profile, I think we should probably refactor them.